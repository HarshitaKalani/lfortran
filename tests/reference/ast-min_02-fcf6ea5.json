{
    "basename": "ast-min_02-fcf6ea5",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/errors/min_02.f90",
    "infile_hash": "6178043dfd10ebfe36cf9c8d0d54302250e3c3759a536bee7e8ab548",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-min_02-fcf6ea5.stdout",
    "stdout_hash": "d9bcd5941fa52494c62b2ebb3c13cef42b9fbbc7d4c4f7dd307c2b09",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}