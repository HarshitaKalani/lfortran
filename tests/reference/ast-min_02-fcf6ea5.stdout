(TranslationUnit
    [(Program
        min_02
        ()
        []
        []
        [(Declaration
            (AttrType
                TypeCharacter
                [(()
                8
                Value)]
                ()
                ()
                None
            )
            []
            [(y
            []
            []
            ()
            None
            ())]
            ()
        )
        (Declaration
            (AttrType
                TypeCharacter
                []
                ()
                ()
                None
            )
            []
            [(z
            []
            []
            ()
            None
            ())]
            ()
        )]
        [(Print
            0
            ()
            [(FuncCallOrArray
                min
                []
                [(()
                y
                ()
                0)
                (()
                z
                ()
                0)]
                []
                []
                []
            )]
            ()
        )]
        []
    )]
)
